src/audio/copier/copier_ipcgtw.c

```diff
@@ -250,3 +250,81 @@ void ipcgtw_zephyr_reset(struct comp_dev *dev)
 		comp_warn(dev, "ipcgtw_reset(): no buffer found");
 	}
 }
+
+#if CONFIG_IPC4_GATEWAY
```

Comment:
copier_ipcgtw.c wont be built if the config CONFIG_IPC4_GATEWAY is not selected. So no need to check this again.

Reply:
ok， I just move with same and keep origin, let me remove it.